plugins/trace_api_plugin/trace_api.swagger.yaml

```diff
@@ -43,12 +47,288 @@ paths:
           content:
             application/json:
               schema:
-                oneOf:
-                  - $ref: "https://eosio.github.io/schemata/v2.0/oas/BlockTraceV0.yaml"
-                  - $ref: "https://eosio.github.io/schemata/v2.0/oas/BlockTraceV1.yaml"
+                $ref: "#/component/schema/BLOCK_TRACE"
```

Comment:
This adds v2 block trace support, but removes v0 and v1 support. These should still be supported - see https://github.com/AntelopeIO/leap/blob/release/3.1/plugins/trace_api_plugin/include/eosio/trace_api/trace.hpp

Reply:
Good point, added all the BlockTraces, and created a new BlockTraceV3 sub-schema